File: Data Structures & Algorithms/evaluate-reverse-polish-notation/submission-0.js
class Solution {
    /**
     * @param {string[]} tokens
     * @return {number}
     */
    evalRPN(tokens) {
        let stack=[];
        
        for(let token of tokens )
        {
             if(token==="+"||token==="-"||token==="*"||token==="/")
             {
                  let num2= stack.pop(); 
                  let num1= stack.pop();
                  let result;

                    switch (token) 
                    {
                        case "+":
                            result = num1 + num2;
                            break;
                        case "-":
                            result = num1 - num2;
                            break;
                        case "*":
                            result = num1 * num2;
                            break;
                        case "/":
                        
                            result = Math.trunc(num1 / num2);
                            break;
                    }
               stack.push(result)
             }
             else
             {
                 stack.push(parseInt(token))
             }

        }

        return stack.pop()
    }
}

File: Data Structures & Algorithms/number-of-one-bits/submission-0.java
class Solution {
    public int hammingWeight(int n) 
    { 
        int result=0;
        for(int i=0;i<32;i++)
        {
            int mask = 1<<i;
            int res = mask & n;
            if(res!=0)
            {
                result++;
            }

        }
        return result;
    }
}
